aten/src/ATen/native/quantized/cpu/OnednnUtils.h

```diff
@@ -5,7 +5,9 @@
 #include <ATen/Tensor.h>
 #include <ATen/native/quantized/PackedParams.h>
 #include <ideep.hpp>
+#ifdef USE_FBGEMM
```

Comment:
Why do we need `USE_FBGEMM` here?

Reply:
`should_use_onednn_quant` uses cpuinfo, which is not supported on s390x. However the function is only invoked for instances where the FBGEMM and OneDNN are both eligible to be used for quantization. As of right now, that only occurs if the QEngine is x86.
The alternative approach, as I've seen in other instances that require `cpuinfo.h` is to instead guard it with
```c++
#if !(defined(__s390x__) || defined(__powerpc__))
#include <cpuinfo.h>
#endif
```

Reply:
I see. It makes more sense to me to use `#if !defined(__s390x__) && !defined(__powerpc__)`. Can you change it to that?